api_loader.go

```diff
@@ -555,6 +555,15 @@ func processSpec(referenceSpec *APISpec,
 	return &chainDef
 }
 
+type DummyProxyHandler struct {
```

Comment:
Why did you decide to separate DummyProxyHandler from ProxyHandler definition? Aren't they related?